src/components/TodoList/TodoInput.js

```diff
@@ -0,0 +1,46 @@
+import { $, isEmptyObject } from '../../utils/utils.js';
+import { DOM_ID, KEY } from '../../constants/constants.js';
+import TodoState from '../../store/todoState.js';
+import { todoListService } from '../../api/todolist.js';
+
+export default class TodoInput {
+  constructor({ userState }) {
+    this.$target = $(DOM_ID.TODO_INPUT);
+
+    this.todoState = TodoState;
+    this.userState = userState;
+
+    this._addEvent();
+  }
+
+  _addEvent() {
+    this.$target.addEventListener('keyup', this._addTodo.bind(this));
```

Comment:
```suggestion
    this.$target.addEventListener('keypress', this._addTodo.bind(this));
```
한글을 입력하고 `Enter`를 눌렀을 때 Todo Item이 2번 입력이 되는 현상이 있었어요!
그래서 원인을 찾아보다 보니 브라우저에 따라서 현상이 다를 것 이란 글을 확인해서 `keyup`이벤트를 `keypress` 이벤트로 대체했을 때 크롬에서 현상이 사라지는 걸 확인했어요.
해당 현상과 관련된 글은 이 [링크](https://github.com/vuejs/vue/issues/10277)를 참고했습니다.

Reply:
아 그렇군요!! 좋은 피드백 감사합니다!!

한글 입력시 "Enter" 까지 처리되서 빈문자일 경우 아무처리도 안하게 했는데 해당 이벤트로 수정하면 그 구문을 제거해도 될 거 같습니다!! ㅎㅎ